src/zig_clang.cpp

```diff
@@ -1592,6 +1592,14 @@ const char* ZigClangVarDecl_getSectionAttribute(const struct ZigClangVarDecl *se
     return nullptr;
 }
 
+bool ZigClangRecordDecl_isPacked(const ZigClangRecordDecl *zig_record_decl) {
```

Comment:
Should this be a more general get attribute?

Reply:
My only nit is that it should be named `getPackedAttribute` to match the other getters.